doc/build.md

```diff
@@ -82,30 +82,12 @@ run all the tests as
 
 ```
 go generate ./...
-go install ./...
```

Comment:
`go install` is only required for releasing Docker image. It is not required for building and testing SQLFlow.